components/NavBar.tsx

```diff
@@ -16,7 +19,17 @@ export default function NavBar(props: Props) {
         key={index}
         className={props.currentPage == page.pageName ? styles.currentPage : ''}
       >
-        <a href={page.pageLink}>{page.pageName}</a>
+        <Button
+          type={ButtonType.NAVBAR}
```

Comment:
Add a new button type named `NAVBAR_ACTIVE` which is equivlent to `[style.navbarBTN, style.active].join()` maybe, the less css in our logic the better imo. and the more templated we have things the better.